cmd/kube-scheduler/app/testing/testserver.go

```diff
@@ -73,7 +73,7 @@ func StartTestServer(t Logger, customFlags []string) (result TestServer, err err
 		if errCh != nil {
 			err, ok := <-errCh
 			if ok && err != nil {
-				klog.ErrorS(err, "Failed to shutdown test server clearly")
+				klog.ErrorS(err, "Failed to shutdown test server clearly: %v", err)
```

Comment:
> > cmd/kube-scheduler/app/testing/testserver.go:76:5: logging function "ErrorS" should not use format specifier "%v" (logcheck)
> 				klog.ErrorS(err, "Failed to shutdown test server clearly: %v", err)
> 				^
> 
> Please review the above warnings. You can
> 

I think that ErrorS already do the formatting, and you don't have to change this

Reply:
Fixed